sw/device/silicon_creator/lib/drivers/address_translation.c

```diff
@@ -0,0 +1,53 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/drivers/address_translation.h"
+#include "sw/device/silicon_creator/lib/base/sec_mmio.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "rv_core_ibex_regs.h"
+
+enum {
+  kBase = TOP_EARLGREY_RV_CORE_IBEX_CFG_BASE_ADDR,
+};
+
+void address_translation_configure(address_translation_slot_t slot,
```

Comment:
One thing that occurs to me is that we will only be configuring slot 0 in the mask ROM and so the slot 1 code path will never be used. Maybe for this reason we should have a function per slot rather than using an enum?

Reply:
done